src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialVideosPageObject.java

```diff
@@ -114,33 +104,47 @@ public void deleteVideo() {
 
   public void verifyDeleteViaGlobalNotifications() {
     YoutubeVideo video = YoutubeVideoProvider.getLatestVideoForQuery("truth");
-
+    System.out.println("Video: " + video.getTitle());
+    System.out.println("Get URL: " + video.getUrl());
     addVideoViaAjax(video.getUrl());
-    deleteVideo();
+    deleteNewestVideo();
     String deletedVideo = "\"File:" + video.getTitle() + "\" has been deleted. (undelete)";
+    System.out.println("BannerNotificationText: " + getBannerNotificationText());
+    System.out.println("Deleted video: " + deletedVideo);
     Assertion.assertEquals(getBannerNotificationText(), deletedVideo);
     PageObjectLogging.log("verifyDeleteVideoGlobalNotifications", "verify video " + deletedVideo
         + " was deleted", true);
   }
 
-  public void verifyDeleteViaVideoNotPresent() {
-    YoutubeVideo video = YoutubeVideoProvider.getLatestVideoForQuery("truth");
+  public boolean isHeaderVisible() {
+    try {
+      wait.forElementVisible(h1Header);
+      return true;
+    } catch (NoSuchElementException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)